### PR TITLE
[i18n:zh_tw] Sync and update translation for zh_TW

### PR DESCRIPTION
**Pull request by andy0130tw**
This patch translates essentially all strings to locale zh_TW, with the following topics excluded:

1. GDPR messages (`gdprMessage` and `gdprMessagePrivacyPolicy`)
2. MetaMask Swaps (keys beginning with "swap")
3. MetaMetrics (keys beginning with "metametrics") -- I am not confident on translating these messages precisely

In addition, a variety of fixes are applied to improve existing translation strings.

- Sync all missing strings from en strings at v9.5.7.
- Update some strings where the corresponding message has changed over time (e.g., `readdToken`).
- Fix some inappropriate punctuations (e.g., missing trailing periods).
- Use more common/agreed terms used in Taiwan (合約、助憶詞、帳戶、函式、...).

I translate these blockchain-related terminologies to the best of my knowledge, but I am open to any adjustments.

### CODE REVIEW

**Comment by andy0130tw** [comment]
I guess I should comment here for the CLA bot to access.

**Comment by andy0130tw** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by mcmire** [comment]
We have a lot of new translations coming in via #13206, so I'll wait until that's merged first.

**Comment by brad-decker** [comment]
@andy0130tw thanks for your patience and apologies for the delayed response. The translation system is now active but we still welcome community translations.If you want to see this landed please rebase and resolve conflicts, then tag me for review. Thank you!

**Comment by andy0130tw** [comment]
I resolved conflicts and rebased my translation against the most recent commit. The PR is ready to be merged (again). Due to the fact that it has been long time since I last worked on this, I patched some small edits I spotted and decided to leave new strings as-is.

File: app/_locales/zh_TW/messages.json
Comment (brad-decker) [inline app/_locales/zh_TW/messages.json:105]: For any keys that are just using the english message please remove those keys, the system falls back to using the english translation when not present for the language.

**Review by brad-decker — CHANGES_REQUESTED** [review]

File: app/_locales/zh_TW/messages.json
Comment (andy0130tw) [reply app/_locales/zh_TW/messages.json:105]: OK. I will remove these keys and rebase again.

Comment (andy0130tw) [reply app/_locales/zh_TW/messages.json]: I have removed redundant keys accordingly.

**Review by brad-decker — APPROVED** [review]

**Comment by brad-decker** [comment]
Thanks @andy0130tw :D -- approved and will seek an additional approval for merge.

**Review by segun — APPROVED** [review]

**Pull request merged**
